packages/vega-util/index.d.ts

```diff
@@ -160,6 +160,6 @@ export const Warn: number;
 export const Info: number;
 export const Debug: number;
 
-export function logger(_?: number, method?: string): LoggerInterface;
+export function logger(_?: number, method?: string, handler?: typeof log): LoggerInterface;
 export function log(...args: readonly any[]): void;
```

Comment:
Looks like the type of the `log` function used by the logger

Reply:
You can add a usage in https://github.com/vega/vega/tree/master/packages/vega-typings/tests/runtime and that will test the typings.

Reply:
Thank you for pointing this out. It's done, along with expliciting these arguments types.